index.html

```diff
@@ -0,0 +1,998 @@
+                the number of refugees admitted into the country to its lowest level in years.
+              </p>
+            </div>
+            <div class="row mx-1"><a href="#"><img src="./images/fire.png" class="img-fluid w-100" alt=""> </a>
+              <a href="#" class="text-body">
+                <h4 class="font-weight-bolder">Refugees Arrive in U.S. Under Lowest Admission Cap in Years</h4>
+              </a>
+              <p class="nowrap">
+                <br>
+                Refugees began arriving in the U.S. after a month-long hiatus during which the Trump administration cut
+                the number of refugees admitted into the country to its lowest level in years.
+              </p>
+            </div>
+            <div class="row mx-1"><a href="#"><img src="./images/sponsor3.png" class="img-fluid w-100" alt=""> </a>
+              <a href="#" class="text-body">
+                <h4 class="font-weight-bolder">Refugees Arrive in U.S. Under Lowest Admission Cap in Years</h4>
+              </a>
+              <p class="nowrap">
+                <br>
+                Refugees began arriving in the U.S. after a month-long hiatus during which the Trump administration cut
+                the number of refugees admitted into the country to its lowest level in years.
+              </p>
+            </div>
+            <div class="row mx-1"><a href="#"><img src="./images/sponsor1.png" class="img-fluid w-100" alt=""> </a>
+              <a href="#" class="text-body">
+                <h4 class="font-weight-bolder">Refugees Arrive in U.S. Under Lowest Admission Cap in Years</h4>
+              </a>
+              <p class="nowrap">
+                <br>
+                Refugees began arriving in the U.S. after a month-long hiatus during which the Trump administration cut
+                the number of refugees admitted into the country to its lowest level in years.
+              </p>
+            </div>
+            <div class="row mx-1 sticky-top two"><a href="#"><img src="./images/sponsor2.png" class="img-fluid w-100"
+                  alt=""> </a>
+              <a href="#" class="text-body">
+                <h4 class="font-weight-bolder">Refugees Arrive in U.S. Under Lowest Admission Cap in Years</h4>
+              </a>
+              <p class="nowrap">
+                <br>
+                Refugees began arriving in the U.S. after a month-long hiatus during which the Trump administration cut
+                the number of refugees admitted into the country to its lowest level in years.
+              </p>
+            </div>
+          </div>
+
+
+          <div class="col-12 col-md-6 col-lg-5 py-2 order-1 order-md-2 ">
+            <div class="col-12  mx-sm-1 two ">
+              <h6 class="font-weight-bolder text-muted">FEATURED STORIES</h6>
+              <img src="./images/bill-taylor-transcript-released.jpg" class="img-fluid w-100" alt="">
```

Comment:
In line 125 should be "TOP STORY"
<h6 class="font-weight-bolder text-muted">TOP STORY</h6>